application/helpers/pdf_helper.php

```diff
@@ -98,8 +103,12 @@ function generate_invoice_pdf($invoice_id, $stream = true, $invoice_template = n
         'show_item_discounts' => $show_item_discounts,
         'custom_fields' => $custom_fields,
     );
-
-    $html = $CI->load->view('invoice_templates/pdf/' . $invoice_template, $data, true);
+    
+    if ($invoice->creditinvoice_parent_id) {
+		$html = $CI->load->view('invoice_credit_templates/pdf/' . $invoice_template, $data, true);
```

Comment:
Why is the template placed in ˋinvoice_credit_templatesˋ instead of ˋinvoice_templatesˋ?

Reply:
Personally I like to separate the general invoices from the credit bills. Each one with a different folder.

Reply:
I would like to keep the templates in one folder as long as we do not have 40 templates for separate use cases.